CHANGELOG.md

```diff
@@ -1,3 +1,242 @@
+  ([#1010](https://github.com/kinvolk/lokomotive/pull/1010)).
+- pkg/assets: cleanup exported API
+  ([#936](https://github.com/kinvolk/lokomotive/pull/936)).
+- Cobra upgrade to v1.1.1
+  ([#1082](https://github.com/kinvolk/lokomotive/pull/1082)),
+  ([#1091](https://github.com/kinvolk/lokomotive/pull/1091)).
+- cli/cmd: cleanups part 2
+  ([#1015](https://github.com/kinvolk/lokomotive/pull/1015)).
+- Add github actions
+  ([#1074](https://github.com/kinvolk/lokomotive/pull/1074)).
+- Makefile: use latest Go when building in Docker
+  ([#1083](https://github.com/kinvolk/lokomotive/pull/1083)).
+- cli/cmd: cleanups part 3
+  ([#1018](https://github.com/kinvolk/lokomotive/pull/1018)).
+- Add new CI config for Packet based FLUO testing
+  ([#1110](https://github.com/kinvolk/lokomotive/pull/1110)).
+
+### Upgrading from v0.4.1
+
+#### Configuration syntax changes
+
+There have been some minor changes to the configurations of worker nodes, `labels` and `taints` format changed from `string` to `map(string)` for AWS and Packet platform.
+
+##### old:
+
+```
+labels = "testing=true"
+
+taints = "nodeType=storage:NoSchedule"
+```
+
+##### new:
+
+```
+labels = {
+  "testing" = "true"
+}
+
+taints = {
+  "nodeType" = "storage:NoSchedule"
+}
+```
+
+#### Cluster upgrade steps
+
+Due to [Terraform](https://github.com/kinvolk/lokomotive/pull/824) and [Kubernetes](https://github.com/kinvolk/lokomotive/pull/1030) upgrades to v0.13+ and v1.19.3 respectively, there are some additional steps that you need to follow. Go to your cluster's directory and follow these steps:
+
+
+1. Upgrade local Terraform binary to version v0.13.X. You can follow the [following guide](https://learn.hashicorp.com/tutorials/terraform/install-cli.) to do that.
+
+2. Export your platform name and assets directory name used in your platform configuration. It will be used in next steps:
+  ```sh
+  export PLATFORM="packet" && export ASSETS_DIR="assets"
+  ``` 
+
+3. Remove old asset files:
+  ```sh
+  rm $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/require.tf \
+  $ASSETS_DIR/terraform-modules/$PLATFORM/flatcar-linux/kubernetes/workers/require.tf \
+  $ASSETS_DIR/terraform-modules/dns/route53/require.tf 2> /dev/null
+  ```
+
+4. Go to terraform directory:
+  ```sh
+  cd $ASSETS_DIR/terraform
+  ```
+
+5. Replace old providers:
+  ```sh
+  terraform state replace-provider -auto-approve registry.terraform.io/-/ct registry.terraform.io/poseidon/ct && \
+  terraform state replace-provider -auto-approve registry.terraform.io/-/template registry.terraform.io/hashicorp/template
+  ```
+
+6. **NOTE:** delete old ReplicaSet `kube-apiserver` only for single node cluster.
+  ```sh
+  kubectl delete rs $(kubectl get rs -n kube-system -l k8s-app=kube-apiserver --no-headers=true  | sort -k5 -h | head -n +2 | awk '{print $1}')
+  ```
+
+7. FelixConfiguration to be managed by Helm:
+  ```sh
+  kubectl label FelixConfiguration default app.kubernetes.io/managed-by=Helm && \
```

Comment:
@iaguis if you have run it more then one time it will throw an error. It works fine for the first time, which should work I think. The error means it's already updated. It works fine, please check the screenshot

![Screenshot 2020-10-27 at 11 08 36 AM](https://user-images.githubusercontent.com/24803604/97261512-fe67f280-1844-11eb-82c7-3a7d2db7cc85.png)

Reply:
^ that. But `--overwrite=true` would not harm either I suppose to gain idempotence.

Reply:
cool, updated